FIX #22 - Add pip description

arielmiki opened the pull request:
Closes #22 

Summary of changes:

Please delete the text below before submitting your contribution.

---

Thanks for contributing!  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.    

---

arielmiki committed:
Add pip description

brownsarahm committed:
fix typo

brownsarahm reviewed (changes requested):
please add an example install maybe, sphinx?

arielmiki committed:
Add example for pip description

brownsarahm committed:
typos, and detail request

add some templating to guide additinal chnges requested and fix typo/langauge errors

brownsarahm reviewed (changes requested):
Thanks for your contribution.  I committed some more templating to  guide what would be helpful in this section. 

this section should also have an exercise.  what could learners in a hands-on workshop do (alone or in small groups) to reinforce the concept of what pip is and how to use it?  

are there common problems that people face with Pip learners should know about how to troubleshoot?
